sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/tests/QuestionAnsweringProjectsClientTests.cs

```diff
@@ -0,0 +1,321 @@
+    {
+        public Uri Endpoint => new("https://leithyta-sdktesting.cognitiveservices.azure.com", UriKind.Absolute);
+
+        public QuestionAnsweringProjectsClient Client => new(Endpoint, new AzureKeyCredential("5d757e79a27f40f896d1dc0c04fb3d09"));
+
+        // private String SampleProjectName = "sampleProjectName";
+
+        // private String SampleDeploymentName = "sampleDeploymentName";
+
+        [Test]
+        public void QuestionAnswerProjectsClientEndpointNull()
+        {
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(
+                () => new QuestionAnsweringProjectsClient(null, null));
+            Assert.AreEqual("endpoint", ex.ParamName);
+        }
+
+        [Test]
+        public void QuestionAnsweringProjectsClientCredentialNull()
+        {
+            Uri endpoint = new Uri("https://test.api.cognitive.microsoft.com", UriKind.Absolute);
+
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(
+                () => new QuestionAnsweringProjectsClient(endpoint, null));
+            Assert.AreEqual("credential", ex.ParamName);
+        }
+
+        [Test]
+        public void GetDeploymentsNull()
+        {
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(() => Client.GetDeployments(null));
+            Assert.AreEqual("projectName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentNullException>(async () => await Client.GetDeploymentsAsync(null).ToEnumerableAsync());
+            Assert.AreEqual("projectName", ex.ParamName);
+        }
+
+        [Test]
+        public void GetSynonymsNull()
+        {
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(() => Client.GetSynonyms(null));
+            Assert.AreEqual("projectName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentNullException>(async () => await Client.GetSynonymsAsync(null).ToEnumerableAsync());
+            Assert.AreEqual("projectName", ex.ParamName);
+        }
+
+        [Test]
+        public void GetSourcesNull()
+        {
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(() => Client.GetSources(null));
+            Assert.AreEqual("projectName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentNullException>(async () => await Client.GetSourcesAsync(null).ToEnumerableAsync());
+            Assert.AreEqual("projectName", ex.ParamName);
+        }
+
+        [Test]
+        public void GetQnasNull()
+        {
+            ArgumentException ex = Assert.Throws<ArgumentNullException>(() => Client.GetQnas(null));
+            Assert.AreEqual("projectName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentNullException>(async () => await Client.GetQnasAsync(null).ToEnumerableAsync());
+            Assert.AreEqual("projectName", ex.ParamName);
+        }
+
+        // These tests should be activated once the bug with the validation generation is fixed.
```

Comment:
Could you create a tracking bug and update the comment with the link as well? That's what we typically do so these TODO (or TODO-like comments: also recommend adding a "TODO" at the beginning since some tools like Visual Studio will actually find and display those in a task list) comments to make sure they aren't lost.

Reply:
Done! I also added TODOs and links for the other issues that we are tracking (transforms and .Core project reference)